docs/basic-guide/database-table-design.mdx

```diff
@@ -80,170 +80,170 @@ ORDER BY expr
+Use FixedString when the difference between the minimum and maximum length of String does not exceed 8, because String stores 8 bytes more offset in memory than FixedString `SELECT min(length(col)), max(length(col)) FROM table`
 
-### Nullable 选择
+### Nullable selection
 
-如果确定列中不包含 Null，不要使用 Nullable 类型，会对性能有负面影响
+If you are sure that the column does not contain Null, do not use Nullable type, it will have a negative impact on performance
 
-### LowCardinality
+### Low Cardinality
 
-如果某个列的基数较低，例如一个 DataPart 内只有不超过 10000 个不相等的值，可以考虑用 LowCardinality 类型。LowCardinality 类型会对原始列进行字典编码。对很多应用来说，处理字典编码的数据可以显著的增加查询速度，并且降低存储空间，提升 IO 效率。
+If the cardinality of a column is low, for example, there are no more than 10,000 unequal values in a DataPart, you can consider using the LowCardinality type. The LowCardinality type dictionary-encodes the original column. For many applications, processing dictionary-encoded data can significantly increase query speed, reduce storage space, and improve IO efficiency.
 
-### 列 Codec 选择
+### Column Codec Selection
 
-通用编码 lz4 / lz4hc / zstd / deflate_qpl
+Generic encoding lz4/lz4hc/zstd/deflate_qpl
 
 [https://clickhouse.com/docs/en/sql-reference/statements/create/table/#general-purpose-codecs](https://clickhouse.com/docs/en/sql-reference/statements/create/table/#general-purpose-codecs)
 
-通常来说 lz4 编码快但压缩率没有 zstd 高, 在支持 Intel IAA 的平台上可以考虑使用 qpl
+Generally speaking, lz4 encoding is fast but the compression rate is not as high as zstd. You can consider using qpl on platforms that support Intel IAA
 
-特殊编码可以和上面的通用编码一起使用: Delta / DoubleDelta / Gorilla / FPC / T64
+Special encodings can be used together with the general encodings above: Delta / DoubleDelta / Gorilla / FPC / T64
 
 [https://clickhouse.com/docs/en/sql-reference/statements/create/table/#specialized-codecs](https://clickhouse.com/docs/en/sql-reference/statements/create/table/#specialized-codecs)
 
-一般来说时序可以用 doubledelta, 随时序变化的值可以用 gorilla, FPC 在 64bit 浮点数上效果比较好. T64 是寻找 64 个值的共同高位进行裁剪，只记录变化的部分
+Generally speaking, doubledelta can be used for timing, and gorilla can be used for values that change with time sequence. FPC works better on 64bit floating point numbers. T64 is to find the common high bits of 64 values for clipping, and only record the changed part
 
-下面是一些最佳实践
+Here are some best practices
 
-- DateTime 类型: 未排序，建议用 Delta+LZ4
-- DateTime 类型：排序，建议用 LZ4
-- Date 类型：未排序，建议用 ZSTD
-- Date 类型：排序（例：PK），建议用 LZ4
+- DateTime type: unsorted, it is recommended to use Delta+LZ4
+- DateTime type: sorting, it is recommended to use LZ4
+- Date type: unsorted, ZSTD is recommended
+- Date type: sorting (eg: PK), it is recommended to use LZ4
 
-### 参考文档
+### Reference Documentation
 
 [https://clickhouse.com/docs/zh/engines/table-engines/mergetree-family/mergetree#tiao-shu-suo-yin-fen-duan-hui-zong-suo-yin-shi-yan-xing-de](https://clickhouse.com/docs/zh/engines/table-engines/mergetree-family/mergetree#tiao-shu-suo-yin-fen-duan-hui-zong-suo-yin-shi-yan-xing-de)
```

Comment:
Need to use the English link, "https://clickhouse.com/docs/en/engines/table-engines/mergetree-family/mergetree#table_engine-mergetree-data_skipping-indexes"